cpp/src/io/json/write_json.cu

```diff
@@ -581,14 +581,13 @@ std::unique_ptr<column> make_column_names_column(host_span<column_name_info cons
 
 void write_chunked(data_sink* out_sink,
                    strings_column_view const& str_column_view,
-                   std::string const& line_terminator,
+                   string_scalar const& d_line_terminator,
                    json_writer_options const& options,
                    rmm::cuda_stream_view stream,
                    rmm::mr::device_memory_resource* mr)
 {
   CUDF_EXPECTS(str_column_view.size() > 0, "Unexpected empty strings column.");
 
-  string_scalar d_line_terminator{line_terminator};
```

Comment:
So this was unused before?

Reply:
It was used before too. Instead of passing `std::string` as argument, now `string_scalar` is passed.